src/NuGet.Core/NuGet.Packaging/Signing/Verification/SignedPackageVerifierSettings.cs

```diff
@@ -180,13 +202,13 @@ public sealed class SignedPackageVerifierSettings
                 allowUnsigned: false,
                 allowIllegal: false,
                 allowUntrusted: false,
-                allowUntrustedSelfIssuedCertificate: true,
                 allowIgnoreTimestamp: false,
                 allowMultipleTimestamps: true,
                 allowNoTimestamp: true,
                 allowUnknownRevocation: true,
                 allowNoRepositoryCertificateList: true,
                 allowNoClientCertificateList: true,
+                alwaysVerifyCountersignature: true,
```

Comment:
Not sure about this default.

Reply:
This specifies that the command line should always verify countersignature.